nginx-controller/controller/controller.go

```diff
@@ -681,12 +751,30 @@ func (lbc *LoadBalancerController) syncIng(task Task) {
 		glog.V(2).Infof("Adding or Updating Ingress: %v\n", key)
 
 		ing := obj.(*extensions.Ingress)
+
+		if _, exists := ing.Annotations["nginx.org/mergible-ingress-type"]; exists {
+			mergableIngExs, err := lbc.createMergableIngresses(*ing)
+			if err != nil {
+				lbc.syncQueue.requeueAfter(task, err, 5*time.Second)
+				lbc.recorder.Eventf(ing, api_v1.EventTypeWarning, "Rejected", "%v was rejected: %v", key, err)
+				return
+			}
+			err = lbc.cnf.AddOrUpdateMergableIngress(mergableIngExs)
```

Comment:
I think we need to emit events for all minions as well

Reply:
^^

Reply:
Done